src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -493,3 +498,28 @@ def list(self):
             "omg/this/is/a/long/path/to/a/file",
             "filterme",
         ]
+
+
+class assert_obj_method_called(object):
```

Comment:
wraps round an object method, allowing us to track whether or not it has been called whilst permitting the original method to be run.